src/emulator/gxm/src/gxm.cpp

```diff
@@ -607,20 +642,23 @@ GLenum translate_internal_format(SceGxmTextureFormat src) {
 
     switch (src) {
         case SCE_GXM_TEXTURE_FORMAT_U8U8U8U8_ABGR:
+        case SCE_GXM_TEXTURE_FORMAT_U8U8U8_BGR:
             return GL_RGBA8;
+        case SCE_GXM_TEXTURE_FORMAT_U4U4U4U4_ABGR:
+            return GL_RGBA4;
         case SCE_GXM_TEXTURE_FORMAT_U8_R111:
+        case SCE_GXM_TEXTURE_FORMAT_U8_111R:
+        case SCE_GXM_TEXTURE_FORMAT_U8_1RRR:
+            // TODO: this is inaccurate
```

Comment:
I think OpenGL uses intensity as the colour palette index, so I think it's OK?

Reply:
Colour palette index? Paletted formats are handled above, with `GL_RGBA`.
`GL_INTENSITY8` is wrong for  these formats because it doesn't return the max value for the channel "1", it just returns this one value ("R") everywhere.

Reply:
Ahh you're right, sorry.